File: Data Structures & Algorithms/sort-an-array/submission-0.py
class Solution:
    def sortArray(self, nums: list[int]) -> list[int]:
        res = [nums[0]]
        for i in range(1,len(nums)):
            inserted = False
            for j in range(len(res)): 
                if res[j] == nums[i] or nums[i] < res[j]:
                    res.insert(j, nums[i])
                    inserted = True
                    break
            if not inserted:
                res.append(nums[i])

        return res
        

File: Data Structures & Algorithms/subarray-sum-equals-k/submission-0.java
class Solution {

    public int subarraySum(int[] nums, int k) {
        int res = 0;

        for (int i = 0; i < nums.length; i++) {
            int count = nums[i];
            if (count == k) {
                res++;
            }
            for (int j = i + 1; j < nums.length; j++) {
                count += nums[j];
                if (count == k) {
                    res++;
                }
            }
        }
        return res;
    }
}